### ISSUE TITLE
Remove JPs keys from GitOpsCI app

### ISSUE THREAD

**Issue by StevenBarre** [BCDevOps/developer-experience#2737]
**Describe the issue**
The GitOpsCI app pushes code from the CCM gen repo to the config repos. It appears to still be using private keys from Justin Pye as his name shows up in emails from GitHub.

**Additional context**
https://github.com/bcgov-c/platform-gitops-gen

**Definition of done**
CCM operates with only keys controlled by current Platform Admins

**Comment by IanKWatts** [comment]
If there were a "GitOpsCI" user, then we could log in as that user, create a new key, and remove the old one.  RoboGofer and RoboWitness are GitHub users, so we could do that with them, and maybe we should next time we have a good opportunity to replace their keys.  But for GitOpsCI, which is a GitHub App, but not a user, an organization administrator has to create the key and it will show as being added by whatever person added it.
Long story short, closing this because there is nothing to do
